src/UpdatePoCommand.php

```diff
@@ -43,6 +43,10 @@ public function __invoke( $args, $assoc_args ) {
 			$destination = $args[1];
 		}
 
+		if ( ! is_file( $destination ) && ! is_dir( $destination ) ) {
```

Comment:
Nit: `file_exists` checks whether a file _or directory_ exists, so two checks in one. So we should be able to use:

```suggestion
		if ( ! file_exists( $destination ) ) {
```